test/source/tests/unit-node.ts

```diff
@@ -1291,19 +1353,18 @@ ZAvn6PBX7vsaReOVa2zsnuY5g70xCxvzHIwR94POu5cENwRtCkrppFnISALpQ1kA
       expect(result.get('Sign/Verify test')).to.equal('[-] skipped, not fully decrypted');
       expect(result.get('SK 0 > LongId')).to.equal('[-] 0485D618EAA64B05');
       expect(result.get('SK 0 > Created')).to.equal('[-] 1606140328 or 2020-11-23T14:05:28.000Z');
-      expect(result.get('SK 0 > Algo')).to.equal('[-] rsa_encrypt_sign');
+      expect(result.get('SK 0 > Algo')).to.equal('[-] rsaEncryptSign');
       expect(result.get('SK 0 > Verify')).to.equal('[-] OK');
-      expect(result.get('SK 0 > Subkey tag')).to.equal('[-] 7');
+      expect(result.get('SK 0 > Subkey object type')).to.equal('[-] SecretSubkeyPacket');
```

Comment:
There might be a way to restore the original behavior using `constructor::tag`.
I haven't tried it yet, should I, @tomholub ?

Reply:
this looks good as is